documentation/01-Prerequisites.md

```diff
@@ -329,7 +323,7 @@ TODO
     pip install -r source\Mlos.Python\requirements.txt
     ```
 
-### Windows Docker Install
+### Option 3: Windows Docker Install
```

Comment:
This wouldn't be option 3, but it also isn't really a separate option at this point.

If you still docker on Windows to do the build you're really targeting a Linux build environment, and so that should be somewhere up above.

Docker is also used to package the python/sqlserver dependencies for the optimizer environment in one of the end-to-end process workflow examples.  It isn't necessary for build in that context though.

There have been separate discussions about relaxing those requirements, but it is there atm.